lib/RexStanUserConfig.php

```diff
@@ -51,11 +51,10 @@ public static function save(int $level, array $paths, array $includes, int $phpV
 
     public static function isBaselineEnabled(): bool
     {
-        $includes = self::getConfig()['includes'];
-        $baselineFile = RexStanSettings::getAnalysisBaselinePath();
+        $includes = self::getPhpStanConfig()['includes'];
 
         foreach($includes as $include) {
-            if ($include === $baselineFile) {
```

Comment:
this comparison did no longer match because we compare relative with absolute paths